Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -393,6 +393,14 @@ namespace gdjs {
     getTextureHeight() {
       return this._textureHeight;
     }
+
+    destroy() {
+      for (const borderSprite of this._borderSprites) {
+        borderSprite.destroy({ texture: true });
+      }
+      this._centerSprite.destroy({ texture: true });
+      this._wrapperContainer.destroy({ children: true });
```

Comment:
I feel that every time we call destroy with params (true, false, texture, children...) we should add a comment explaining why we pick those values.
I'm not at ease with modifying this in the future without knowing why we picked true or false for any of those parameters

Reply:
Agree, let's explain this particularly "complex" destroy method.